content/docs/0.6.0/reference/manifest_installation/index.md

```diff
@@ -68,29 +68,31 @@ kubectl apply -f https://raw.githubusercontent.com/keptn/keptn/$KEPTNVERSION/ins
 
 ## Summary of applied images
```

Comment:
Can we please make this section a subsection of `https://deploy-preview-558--keptn.netlify.app/docs/0.6.0/reference/manifest_installation/#setup-keptn-quality-gates`

Reply:
It makes totally sense! I changed it to a subsection.